cli/src/helper.rs

```diff
@@ -25,30 +33,70 @@ pub fn get_password(prompt: &str, confirmation: bool) -> Result<String, Error> {
     Ok(password.interact()?)
 }
 
-pub async fn pick_account(wallet: &Wallet) -> Result<Option<u32>, Error> {
-    let accounts = wallet.get_accounts().await?;
+pub fn get_decision(prompt: &str) -> Result<bool, Error> {
+    loop {
+        let input = Input::<String>::new()
+            .with_prompt(prompt)
+            .default("yes".into())
+            .interact_text()?;
+
+        match input.to_lowercase().as_str() {
+            "yes" | "y" => return Ok(true),
+            "no" | "n" => return Ok(false),
+            _ => {
+                println_log_error!("Accepted input values are: yes|y|no|n");
+            }
+        }
+    }
+}
+
+pub async fn get_account_name(prompt: &str, wallet: &Wallet) -> Result<String, Error> {
+    let accounts = wallet.get_account_aliases().await?;
+    loop {
+        let input = Input::<String>::new().with_prompt(prompt).interact_text()?;
+        if input.is_empty() || !input.is_ascii() {
+            println_log_error!("Invalid input, please choose a non-empty name consisting of ASCII characters.");
```

Comment:
Is it even possible to create an empty one? Don't think Input lets you do it

Reply:
you're right, but I didn't want to rely on that if they decide to change that behavior. Easy to forget adding this check then.

Reply:
Hmm ok